birdvoxpaint/indices.py

```diff
@@ -1,37 +1,32 @@
 import librosa
 import numpy as np
+from scipy import stats
 
 
 def acoustic_complexity_index(S):
-    spectrogram = np.abs(S)**2
-    flux = np.abs(np.diff(spectrogram, axis=1))
-    return np.sum(flux, axis=1)/np.sum(spectrogram, axis=1)
+    flux = np.abs(np.diff(S, axis=1))
```

Comment:
the problem behind all these changes is that it prevents us to define any index that relies on the phase of the STFT in the future, such as phase deviation. i see the appeal in not computing the modulus multiple times, but i would be surprised if that had a noticeable effect on speed.

Reply:
Good point! I forget abt phase sometimes haha. I just saw it in every function and instinctually cleaned it up, but I'm happy to undo it